central/main.go

```diff
@@ -747,6 +751,12 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 			ServerHandler: clustersHelmConfig.Handler(clusterDataStore.Singleton()),
 			Compression:   true,
 		},
+		{
+			Route:         "/api/product/usage/secured-units/csv",
```

Comment:
What does `secured-units` mean?

Reply:
Nodes and CPU Units of the secured clusters.